AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1953,7 +1953,7 @@ dobro.mail.ru##.dobro-popup-subscribe
 ! ##.newsletter
-americanthinker.com,mindtools.com,abw.by,firstthings.com,arbeitsrecht.org,sheknows.com,propublica.org,paquetaesportes.com.br,carandbike.com,softwarelicense4u.com,ginx.tv,oggusto.com,brocardi.it,mjmultimedia.se,esteban.fr,petri.com,expressio.fr,vice.com,kliinik.ee,blog.jetbrains.com,thenextweb.com,tipy.interia.pl,folhadoslagos.com,peoplesworld.org,express.co.uk,enerjiportali.com,vodafone.ua,ciapc.com.br,dontpayfull.com,salutbonjour.ca,coffeecup.com,overkillshop.com,caffeborbone.it,addictivetips.com,anwalt.de,arukereso.hu,bilan.ch,boehms-dax-strategie.de,catwalkyourself.com,elle.de,freundin.de,hgtv.ca,instyle.de,investopedia.com,itstep.dp.ua,kimintelefonu.com,macrumors.com,onygo.com,ria.com,thewirecutter.com,vulture.com,wikitribune.com##.newsletter
+ziarulevenimentul.ro,americanthinker.com,mindtools.com,abw.by,firstthings.com,arbeitsrecht.org,sheknows.com,propublica.org,paquetaesportes.com.br,carandbike.com,softwarelicense4u.com,ginx.tv,oggusto.com,brocardi.it,mjmultimedia.se,esteban.fr,petri.com,expressio.fr,vice.com,kliinik.ee,blog.jetbrains.com,thenextweb.com,tipy.interia.pl,folhadoslagos.com,peoplesworld.org,express.co.uk,enerjiportali.com,vodafone.ua,ciapc.com.br,dontpayfull.com,salutbonjour.ca,coffeecup.com,overkillshop.com,caffeborbone.it,addictivetips.com,anwalt.de,arukereso.hu,bilan.ch,boehms-dax-strategie.de,catwalkyourself.com,elle.de,freundin.de,hgtv.ca,instyle.de,investopedia.com,itstep.dp.ua,kimintelefonu.com,macrumors.com,onygo.com,ria.com,thewirecutter.com,vulture.com,wikitribune.com##.newsletter
```

Comment:
Yes, I can see it. But what about ads on the right column and on the main page?

Reply:
Shouldn't they be addressed in a Romanian list?

Reply:
They should be, but if we see such a case that language filter doesn't have a solution - we solve it in Base - foreign.txt - rules are added to the necessary language section.

Reply:
What about
```
ziarulevenimentul.ro##.set_mg_of_content_b
ziarulevenimentul.ro##.set_tab_sales
ziarulevenimentul.ro##p > a[target="_blank"] > img
ziarulevenimentul.ro##a[href^="/app/adclick.php?id="]
```

Reply:
@Alex-302 Those rules seem to work.

Reply:
@IThinkIAmCool add them here Base - foreign.txt - Romanian section - 
between those rules with task comment
```
! ziarulevenimentul.ro - ads
ziarulevenimentul.ro##.set_mg_of_content_b
ziarulevenimentul.ro##.set_tab_sales
ziarulevenimentul.ro##p > a[target="_blank"] > img
ziarulevenimentul.ro##a[href^="/app/adclick.php?id="]
``` 

https://github.com/AdguardTeam/AdguardFilters/blob/8e21a01065ca7ee758c086f95a0149c320745f05/EnglishFilter/sections/foreign.txt#L7411
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/125033919-a30b2700-e098-11eb-8e6d-9dc05a55f998.png)
</details><br/>